src/data-access/repositories/mongo/mongo-status-message-repository.ts

```diff
@@ -30,6 +30,13 @@ export class MongoStatusMessageRepository extends BaseMongoRepository implements
     return await this.getCollection().find<StatusMessage>({}).toArray()
   }
 
+  public async getStatusMessagesWithIdPrefix(idPrefix: string): Promise<StatusMessage[]> {
+    this.assertDatabaseConnection(this.getStatusMessagesWithIdPrefix.name)
+    return await this.getCollection().find<StatusMessage>({
+      id: new RegExp(idPrefix, 'g')
```

Comment:
This seems to check if the id prefix is in the id and not at the start of the id.

Reply:
It checks if it is a prefix now.